src/python/WMCore/Services/RucioConMon/RucioConMon.py

```diff
@@ -0,0 +1,110 @@
+#!/usr/bin/env python
+# coding=utf-8
+"""
+Rucio Consistency Monitor Service class developed on top of the native WMCore
+Service APIs providing custom output and handling as necessary for the
+CMS Workload Management system
+"""
+
+from __future__ import division, print_function, absolute_import
+
+from urllib.parse import urlencode
```

Comment:
Since our code is meant to be dual stack, I'd suggest to have this import fixed with the future standard_library.

Reply:
fixed

Reply:
Not yet, see my other comment please.

Reply:
fixed